plugins/com.google.cloud.tools.eclipse.appengine.deploy/plugin.xml

```diff
@@ -43,4 +43,28 @@
     </handler>
   </extension>
 
+  <extension
+    point="org.eclipse.ui.console.consolePatternMatchListeners">
+    <consolePatternMatchListener
+      class="com.google.cloud.tools.eclipse.ui.util.console.PatternToHyperlinkConverter"
+      id="com.google.cloud.tools.eclipse.appengine.deploy.consolePatternMatchListener"
+      regex="https://.+\.appspot\.com">
+      <enablement>
+        <test
+          property="com.google.cloud.tools.eclipse.ui.util.console.name"
+          value="App Engine Deploy">
```

Comment:
I think this would be better testing the console _type_.  I'd expect the console _name_ to include the projectId + version, and maybe date and time?